src/administration/horizontal_scaling.md

```diff
@@ -63,30 +63,40 @@ Here's a quick example on how you could start 3 web servers, 3 federation server
 ```
 # scheduled tasks
 lemmy_server --disable-http-server --disable-activity-sending
+```
+
+Or run the Docker container with the `-e LEMMY_DISABLE_HTTP_SERVER=true` and `-e LEMMY_DISABLE_ACTIVITY_SENDING=true` ENV parameters.
```

Comment:
Dont put the `-e` part, its not needed if you run natively or via docker-compose so it can be confusing.

Reply:
I intentionally put it as it is the equivalent to the above non-docker direct start option. If you want to explain the docker-compose method, then the equivalent would be to modify a Systemd service file for the non-docker version.

But anyone willing to customize their docker-compose will know the difference anyway.